test/utils/test_generic.py

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env python3
+
+import unittest
+
+import torch
+
+from linear_operator.test.base_test_case import BaseTestCase
+from linear_operator.utils.generic import _to_helper
+
+
```

Comment:
In order to test that this does the right thing for non-cpu tensors, could add a mock tensor like this:
```suggestion
class TestTensor(Tensor):
    @property
    def device(self):
        return self._device
```
Then in the test do 
```python
A = TestTensor()
A._device = torch.device("cuda")
```
`A` will then pretend to be on a CUDA device, without us having to have access to one.

Reply:
Yeah makes sense, or I can just use `mock` directly to assign the attribute.